src/reducers/userReducer.js

```diff
@@ -0,0 +1,50 @@
+const userReducer = (state, action) => {
+        switch (action.type) {
+			case "LOGIN_REQUEST":
+				return (state = {
```

Comment:
Suggestion:- If possible check alignment once in this file. It is useful more readable.

Reply:
Sure